source/frontends/libretro/libretro.cpp

```diff
@@ -36,6 +36,38 @@ namespace
     return std::equal(ending.rbegin(), ending.rend(), value.rbegin());
   }
 
+  void update_variables(void)
```

Comment:
no void. It is only in the code AppleWin code for "legacy" reasons.

Reply:
Applied.